packages/flutter_tools/lib/src/build_system/targets/macos.dart

```diff
@@ -83,111 +125,195 @@ class UnpackMacOS extends Target {
   }
 }
 
-/// Tell cocoapods to re-fetch dependencies.
-class DebugMacOSPodInstall extends Target {
```

Comment:
Why is pod-related code being removed in this PR?

Reply:
There seem to be some pod changes that are incompatible with the desktop project, and this was part of the issue with the last switch to assemble. Since I'm not currently using it and don't plan to in the near future, I've removed it for now.

I could leave it here and clean it up separately though

Reply:
To be clear though, this version is not causing the currently-working pod workflow to not run (as with the previous version)? I'm assuming that's the `processPodsIfNeeded` in build_macos.

Reply:
I've completely removed pods from the assemble code path for now, so it runs the same as it did before.